src/_locales/de/messages.json

```diff
@@ -0,0 +1,254 @@
+{
+  "extensionName": {
+    "message": "Cookie AutoDelete",
+    "description": "Name of the extension."
+  },
+
+  "extensionDescription": {
+    "message": "Beherrsche deine Cookies! Lasse unbenutzte Cookies von deinen geschlossenen Tabs automatisch löschen.",
+    "description": "Control Your Cookies! Auto Delete Unused Cookies From your Closed Tabs While Keeping The Ones You Want."
+  },
+
+  "notificationTitle": {
+    "message": "Cookie AutoDelete: Cookies wurden gelöscht!",
+    "description": "Cookie AutoDelete: Cookies were Deleted!"
+  },
+
+  "notificationContent": {
+    "message": "$Num$ Cookies gelöscht von: $Websites$",
+    "description": "123 Deleted Cookies from: google.com, facebook.com",
+    "placeholders": {
+      "num" : {
+        "content" : "$1",
+        "example" : "123"
+      },
+      "websites" : {
+        "content" : "$2",
+        "example" : "google.com, facebook.com"
+      }
+    }
+  },
+
+  "hostWebsiteText": {
+    "message": "Webseite:",
+    "description": "Host Website:"
+  },
+
+  "cookieCleanupText": {
+    "message": "Cookies aufräumen",
+    "description": "Cookie Cleanup"
+  },
+
+  "cookieCleanupIgnoreOpenTabsText": {
+    "message": "Cookies aufräumen (offene Tabs ignorieren)",
+    "description": "Cookie Cleanup (Ignore Open Tabs)"
+  },
+
+  "clearCookiesForDomainText": {
+    "message": "Lösche alle Cookies dieser Domain",
+    "description": "Clear All Cookies For This Domain"
+  },
+
+  "activeModeSwitchText": {
+    "message": "Aktiver Modus",
+    "description": "Active Mode"
+  },
+
+  "switchToWhiteListText": {
+    "message": "Zur Whitelist hinzufügen?",
+    "description": "Add to WhiteList?"
```

Comment:
Just a nitpick, but I don't like the wording of this… I think "Whitelisted" or something like this makes more sense, since it is a checkbox, no reason for a question. Translated it literally though to stay true to the english source.